openquakeplatform/bin/deploy.sh

```diff
@@ -624,6 +624,7 @@ oq_platform_install () {
     if [ "$GEM_IS_INSTALL" == "y" ]; then
         # Load our users. Default password must be changed
         openquakeplatform loaddata ${oqpdir}/common/fixtures/*.json
+        cat ${oqpdir}/common/fixtures/*.sql | openquakeplatform dbshell
```

Comment:
This will prompt the user for a password that has been generated by the script itself and is not immediate to be found (i.e you have to open a new terminal/ssh and dig into the `local_settings.py`).

We have two solutions:
1. Just echo the password before that line saying "Please enter this password: abcd123"
2. Find a better way to load SQL fixtures

2. can be done placing the SQL in an application `sql` folder with a model name.